src/sections/landing4/Hero.js

```diff
@@ -15,9 +15,9 @@ const SectionStyled = styled(Section)`
 
 const ImgRight = styled(Box)`
   position: relative;
-  left: -100px;
+  left: -200px;

```

Comment:
Can these changes be added to a breakpoint so that the image sizing is a bit more consistent?

Reply:
They have list of breakpoints where is small breakpoint starts with` min-width: 576px`.  So if we put `left` propertie in a device.sm breakpoint we only will have changes from **576px**